examples/tracked-controllers.html

```diff
@@ -48,6 +50,10 @@
         debug: true;
         adapter: wseasyrtc;
     "
+      renderer="antialias: true;
+      colorManagement: true;
+      sortObjects: true;
+      physicallyCorrectLights: true;"
```

Comment:
these make seeing the controller buttons much easier, they are really hard to see without this

Reply:
okay